packages/v4/src/content/design-guidelines/demos/password-strength/password-strength.md

```diff
@@ -0,0 +1,25 @@
+---
+id: Password strength
+section: demos
+---
+
+A **password strength indicator** is displayed to a user after they have entered a password in a field. It allows the user to understand the strength of a password after it has met the password requirements. The strength of the password is determined by a validation algorithm after the specific rules are met.  Using a password strength indicator allows users to create stronger passwords, offering better protection from breaches. 
+
+## Elements
+
+<img src="./img/password strength indicator weak.png" alt="password strength indicator weak"/>
+<br/>
+<img src="./img/password strength indicator medium.png" alt="password strength indicator medium"/>
+<br/>
+<img src="./img/password strength indicator strong.png" alt="password strength indicator strong"/>
+<br/>
+The password strength indicator is located on the top right corner of the input field. It displays the following strength types:
+- Weak
```

Comment:
Could you change this to a numbered list to match the pointers in the images? so:

1. Weak
2. Medium
3. Strong